### PR TITLE
Cu 88qcp5 css integration show hide button

### PR DESCRIPTION
**Pull request by Hamza-A**
This branch aims to refactor the Show/Hide button on logins page to make it stylized as on the wireframe

## Description
To do this, I had moved the `<ShowMoreButton />` to its own file in the `src/components/display/fewlines` folder, by the way I've created its story.
This component contains a new one `<Triangle />` which is the triangle we can se beside the text on the button in the picture below. i've also created a story for the `<Triangle />`

## How Has This Been Tested?
I only had to change the imports in a test file and add some slight changes to some of them

## Screenshots
image

## Types of changes

- Chore (non-breaking change which refactors / improves the existing code base)

## Checklist:

- :green_circle: My code follows the code style of this project.
- :red_circle: My change requires a change to the documentation.
- :red_circle: I have updated the documentation accordingly.
- :green_circle: I have read the [**CONTRIBUTING**][contributing_file] document.
- :red_circle: I have added tests to cover my changes.
- :green_circle: All new and existing tests passed.

[contributing_file]: https://github.com/fewlinesco/connect/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: src/components/display/fewlines/ShowMoreButton.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+export const StandardShowMoreButton = (): JSX.Element => {
+  const [hide, setHideSecondary] = React.useState<boolean>(true);
+  return (
+    <ShowMoreButton
+      hide={hide}
+      quantity={4}
+      setHideSecondary={setHideSecondary}
+    />
+  );
```
Comment (Fuzznimp) [inline src/components/display/fewlines/ShowMoreButton.stories.tsx:15]:
```suggestion
  const [hide, setHideSecondary] = React.useState<boolean>(true);

  return (
    <ShowMoreButton
      hide={hide}
      quantity={4}
      setHideSecondary={setHideSecondary}
    />
  );
```

File: src/components/display/fewlines/ShowMoreButton.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+};
+
+export const Button = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-bottom: 3rem;
+  &:hover {
+    cursor: pointer;
+  }
+`;
```
Comment (Fuzznimp) [inline src/components/display/fewlines/ShowMoreButton.tsx:33]:
```suggestion
export const Button = styled.div`
  display: flex;
  justify-content: center;
  margin-bottom: 3rem;
  cursor: pointer;
`;
```

Quite unintuitive but you don't need to precise on hover :D.

File: src/components/display/fewlines/Triangle.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+const Rotate = styled.span`
+  .rotate {
+    transform: rotate(180deg);
+    margin-bottom: 0.3rem;
+  }
+
+  .normal {
+    margin-bottom: 0.3rem;
+  }
+`;
```
Comment (Fuzznimp) [inline src/components/display/fewlines/Triangle.tsx:34]: Adding a transition on the `rotate` is probably a good idea :).

You can also handle this without a class, right in your span css.

**Review by Fuzznimp — CHANGES_REQUESTED** [review]

File: src/components/display/fewlines/ShowMoreButton.stories.tsx (hunk 1 above)
Comment (Hamza-A) [reply src/components/display/fewlines/ShowMoreButton.stories.tsx:15]: OK fixed f96f40c

File: src/components/display/fewlines/ShowMoreButton.tsx (hunk 2 above)
Comment (Hamza-A) [reply src/components/display/fewlines/ShowMoreButton.tsx:33]: OK fixed f96f40c

File: src/components/display/fewlines/Triangle.tsx (hunk 3 above)
Comment (Hamza-A) [reply src/components/display/fewlines/Triangle.tsx]: OK fixed f96f40c

**Review by Fuzznimp — APPROVED** [review]

**Pull request merged**